src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -990,7 +991,7 @@ public void updateEntitySetMetadata( UUID entitySetId, MetadataUpdate update ) {
             }
         }
 
-        final var newEntitySet = ( EntitySet ) entitySets.executeOnKey(
```

Comment:
😬

Reply:
Is this a bad face?

Reply:
haha sorry. We don't have clear formatting standards but I prefer the extra spacing style. Super don't need to change this at all but I wanted to flag it cause we should settle on some formatting preferences as a team

Reply:
Oh I see! I think I submitted this before Katherine showed me the formatting style we use. I will update and push that